vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/openid/PresentationFactory.kt

```diff
@@ -73,7 +73,7 @@ internal class PresentationFactory(
             credentialPresentation = credentialPresentation,
         ).getOrElse {
             Napier.w("Could not create presentation", it)
-            throw OAuth2Exception(Errors.USER_CANCELLED, it)
+            throw AccessDenied("Could not create presentation", it)
```

Comment:
was this one of the confusing error messages reported?

Reply:
Yes, this has now been clarified in OpenID4VP, and we're using it here